tests/unit/database/test_image.py

```diff
@@ -89,6 +91,10 @@ def runTest(self):
                                                                'project 1')
         self.assertEqual(img_id, 2)
 
+        with pytest.raises(db_exceptions.ImageNotFoundException):
```

Comment:
Dont use pytest for writing tests, it is being used as a TestRunner only. Use assertRaises instead